.devcontainer/devcontainer.json

```diff
@@ -33,17 +34,15 @@
 		"pmbenjamin.vscode-snyk",
 		"redhat.vscode-yaml",
 		"richie5um2.vscode-sort-json",
-		"tabnine.tabnine-vscode",
 		"usernamehw.errorlens",
 		"visualstudioexptteam.vscodeintellicode",
-		"yzhang.markdown-all-in-one",
-		"github.copilot",
-		"karyan40024.gotmpl-syntax-highlighter"
+		"wenfangdu.jump",
+		"yzhang.markdown-all-in-one"
 	],
 	"postCreateCommand": "notify-dev-entrypoint.sh",
 	"remoteUser": "vscode",
 	"mounts": [
-		"source=${env:HOME}${env:USERPROFILE}/.aws,target=/home/vscode/.aws,type=bind",
-		"source=${env:HOME}${env:USERPROFILE}/.kube,target=/home/vscode/.kube,type=bind"
```

Comment:
I removed `${env:USERPROFILE}`, as this was repeating the same folder as previous and made the devcontainer fail. I am not sure how that worked before. I tested in MacOSX and Windows.